stakers-metrics/src/utils.ts

```diff
@@ -1,32 +1,28 @@
-import { getUrlFromDnpName } from "@dappnode/types";
-import { networks } from "@dappnode/types";
+import { networks, getJsonRpcApiFromDnpName } from "@dappnode/types";
 import axios, { AxiosError } from "axios";
 import logger from "./logger.js"; 
 
-const urls = getUrlFromDnpName();
+/**
+ * Gets the client URL for a given network and type (execution or consensus).
+ * @param network - The network for which the client URL is required.
+ * @param type - The type of client (execution or consensus).
+ * @returns The client URL for the specified network and type, or undefined if not found.
+ */
+export function getClientUrl(network: typeof networks[number], clientType: "execution" | "consensus"): string | undefined {
```

Comment:
Isn't the type `Network` already defined in types?

You should replace this `typeof networks[number]` by `Network`

Reply:
done